analyzer/consensus/consensus.go

```diff
@@ -308,6 +309,14 @@ func (m *Main) processBlock(ctx context.Context, height int64) error {
 		return err
 	}
 
+	for i, b := range queries {
+		if b.Len() == 0 {
+			m.logger.Info(fmt.Sprintf("Block %d missing %d data", height, i))
```

Comment:
`missing 3 data` might be hard to understand

`Block %d goroutine %d emitted zero queries` :tipping_hand_person:

Reply:
Nice, updated!